lib/validation.js

```diff
@@ -9,8 +9,10 @@ const joi = require('joi');
 /**
  * Validate user input against rules
  *
- * @param {function} rule - Rule created with the joi validation framework
- * @returns {Error|boolean} return true if input is valid, return the error otherwise
+ * @param {function} rule - Validation rule created with the joi validation framework
+ * @returns {function} calls an anonymous function that returns either
```

Comment:
It returns an anonymous function, it doesn't call it.
The fact that the underlying function is anonymous is an implementation detail and should not be part of the documentation.